IPython/core/display.py

```diff
@@ -278,6 +280,14 @@ def display(*objs, include=None, exclude=None, metadata=None, transient=None, di
         from IPython.display import display
 
     """
+    # Check whether display() has been called from IPython shell or from Pure Python shell
+    from IPython.core.interactiveshell import InteractiveShell
+    if not InteractiveShell.is_instantiated:
+        # Warn user who are calling display() from Pure Python Shell
+        #TODO:Add standard print() functionality
+        warn('display() no longer supports calls from Pure Python shell. To experience it\'s features use IPython shell instead.')
```

Comment:
I think we might want to have `print(*objs)` directly, no need to warn I think, but that just my opinion.